### PR TITLE
ETOAST-128: Text Area component

### PR DESCRIPTION
**Pull request by suabochica**

### CODE REVIEW

File: src/TextArea/index.tsx [hunk 1]
```diff
@@ -2,25 +2,48 @@ import React from 'react';
...
+
+  id?: string;
+  isDisabled:boolean;
+  name?: string;
   hasIcon?: boolean;
-  textAreaProps?: React.ComponentPropsWithoutRef<"div">;
 }
 
-const TextArea = ({ name = "normal", variant = "outline", hasIcon = false, textAreaProps = {} }: TextAreaProps) => {
+const Padded = ({ children }) => <div style={{margin: "16px"}}> {children} </div>;
```
Comment (john-angel) [inline src/TextArea/index.tsx]: Should this style be part of @e-toast/css?

Comment (suabochica) [reply src/TextArea/index.tsx]: 🤔 Good question I define this style following the [guides](https://github.com/bod-endava/e-toast-ds/blob/5f24b99af78ce8b6617908eecca214fc4db7b19f/packages/react-sandbox/src/App.js#L4-L6) that we have in the react-sandbox package that are at the beginning of the `App.js` file.

Comment (suabochica) [reply src/TextArea/index.tsx]: After check, we can delete this style. Changes updated.

**Comment by john-angel** [comment]
Looks good

**Review by john-angel — APPROVED** [review]
Looks good

**Pull request merged**